src/lambdas/initiateDataRequest/validateZendeskRequest.spec.ts

```diff
@@ -359,6 +359,21 @@ describe('validateZendeskRequest', () => {
     ])
   })
 
+  it(`should handle dataPaths containing empty strings`, async () => {
```

Comment:
Could we add a completely empty string to this? It seems like what you've got here is some double spacing, which is also a good test, but I'd like to make sure we also handle the possibility of nothing going on, when we should get back an empty array.

Reply:
Done (it had to be a separate test rather than iterating over different dataPath values)